utils/removeArraysUniqueValues.js

```diff
@@ -0,0 +1,7 @@
+const removeArraysUniqueValues = (firstArray, secondArray) => {
+  const filteredArray = firstArray.filter((item) => !secondArray.includes(item))
```

Comment:
i think there is no need of additional variable, just return the result
same in ***mergeArraysUniqueValues***

Reply:
fixed